tasks/compile-example.js

```diff
@@ -139,61 +181,42 @@ module.exports = function(config, updateTimestamp) {
         gutil.log('Generated ' + previewFile.relative);
         stream.push(previewFile);
       }
-
-    }
-
-    cb();
-  }
-
-  function endStream(cb) {
-    // no examples passed in, no file goes out
-    if (!latestFile || !examples) {
-      cb();
-      return;
-    }
-
-    const categories = mapToCategories(examples);
-    const stream = this;
-    const args = {
-      categories: categories,
-      title: 'AMP by Example',
-      desc: 'A hands-on introduction to Accelerated Mobile Pages (AMP) ' +
-        'focusing on code and live samples. Learn how to create AMP pages ' +
-        'and see examples for all AMP components.',
-      timestamp: timestamp,
-      fileName: '/'
-    };
-
-    Metadata.add(args);
-    args.fileName = '';
-    const html = templates.render(config.templateIndex, args);
-    const indexFile = latestFile.clone({contents: false});
-    indexFile.path = path.join(latestFile.base, "index.html");
-    indexFile.contents = new Buffer(html);
-    gutil.log('Generated ' + indexFile.relative);
-    stream.push(indexFile);
-    cb();
+    });
   }
 
-  function mapToCategories(examples) {
+  function mapToCategories(examples, currentExample) {
     const categories = [];
     let currentCategory;
-    sort(examples).forEach(function(exampleFile) {
-      // add example to categories instance
-      if (!currentCategory || currentCategory.name != exampleFile.category()) {
-        currentCategory = {
-          name: exampleFile.category(),
-          examples: []
-        };
-        categories.push(currentCategory);
+    sort(examples).filter(exampleFile => !exampleFile.document.metadata.draft)
+    .forEach(function(exampleFile) {
+      if (!exampleFile.document.metadata.draft) {
```

Comment:
remove ` if (!exampleFile.document.metadata.draft) {` as it is no longer needed and handled by the filter above